tests/lib/snaps/test-snapd-daemon-notify/meta/snap.yaml

```diff
@@ -3,8 +3,12 @@ version: 1.0
 summary: Basic daemon-notify snap
 description: A basic snap which allow sending notification messages to systemd through the notify socket
 
+hooks:
+  install:
+    plugs: [network, network-bind]
```

Comment:
Why do we need to use both network and network-bind here?

Reply:
Eh, whenever I use `network` I always also add `network-bind` as a matter of taste, but it's not strictly necessary

Reply:
Removed